unit_tests/src/test_access_time_queue.cpp

```diff
@@ -97,6 +97,17 @@ TEST_CASE("invalid initialization")
         REQUIRE_THROWS_AS([] { atq::init("", 1); }(), irods::exception);
     }
 
+    SECTION("queue name prefix starts with character other than alphabet or underscore")
```

Comment:
alphabet -> alphabetic

Reply:
alphanumeric

Reply:
> alphanumeric

The access time queue will reject names starting with a number.

Updated.